roles/lsaai/tasks/main.yml

```diff
@@ -1,4 +1,29 @@
 ---
+- name: "Get the list of namingContexts from all databases"
+  community.general.ldap_search:
+    server_uri: "ldapi://%2Fvar%2Frun%2Fslapd%2Fldapi"
+    dn: "olcDatabase={% raw %}{{% endraw %}{{ item }}{% raw %}}{% endraw %}mdb,cn=config"
+  loop: "{{ range(1, ldap_domains | community.general.json_query('*.ldap_db_index') | max + 1) }}"
+  failed_when: false # it's not an error if database does not exist
```

Comment:
A missing database may not be an error, but there are many other ways this ldap_search may fail and which should be treated as an error that halts the execution of the play.